scripts/globals/status.lua

```diff
@@ -2591,7 +2581,7 @@ dsp.animation =
     -- 63 through 72 are used with /sitchair
     -- 73 through 83 sitting on air (guessing future use for more chairs..)
     MOUNT                   = 85,
-    -- TRUST                = 90, -- This is the animation for a trust NPC spawning in.
+    TRUST                   = 90, -- This is the animation for a trust NPC spawning in.
```

Comment:
We never actually used this in lua because the core&packet do it

Reply:
i can leave this commented then